RPerformanceTracking/Private/_RPTClassManipulator+UIWebView.m

```diff
@@ -148,7 +148,7 @@ + (void)_swizzleUIWebViewDelegate:(id<UIWebViewDelegate>)delegate
             uint_fast64_t trackingIdentifier = [objc_getAssociatedObject(webView, _RPT_UIWEBVIEW_TRACKINGIDENTIFIER) unsignedLongLongValue];
             if (trackingIdentifier)
             {
-                [_RPTTrackingManager.sharedInstance.tracker end:trackingIdentifier statusCode:httpResponse.statusCode];
+                [_RPTTrackingManager.sharedInstance.tracker updateStatusCode:httpResponse.statusCode trackingIdentifier:trackingIdentifier];
```

Comment:
There is a helper for updating status code`+static void updateStatusCodeForWebView(NSInteger statusCode, WKWebView *webView)` a bit below. Why not to use it here?